### PR TITLE
Bug 1130548: update the base AMIs to work for both servers and builders

### PR DESCRIPTION
**Pull request by djmitche**
I'm in the process of testing this for buildbot masters, and will create a canary for builders tomorrow before landing it.

### CODE REVIEW

File: cloudtools/scripts/aws_create_ami.py [hunk 1]
```diff
@@ -211,15 +211,11 @@ def create_ami(host_instance, args, config, instance_config, ssh_key,
...
             put('etc/yum-local.cfg', '%s/etc/yum-local.cfg' % mount_point)
-            put('groupinstall', '/tmp/groupinstall')
-            put('additional_packages', '/tmp/additional_packages')
         yum = 'yum -d 1 -c {0}/etc/yum-local.cfg -y --installroot={0} '.format(
             mount_point)
-        # groupinstall emulates the %packages section of the kickstart config
-        run('%s groupinstall "`cat /tmp/groupinstall`"' % yum)
-        # and this attempts to emulate the additional packages that Anaconda installs
-        # as it needs them.
-        run('%s install `cat /tmp/additional_packages`' % yum)
```
Comment (rail) [inline cloudtools/scripts/aws_create_ami.py]: Hmmm, I think we have these files (groupinstall and aditional_packages) in other ami configs. Do the new commands install kernel, grub,i etc?

Comment (djmitche) [inline cloudtools/scripts/aws_create_ami.py]: True, this makes `aws_create_ami` unable to create CentOS-6.2 AMIs anymore -- but there's only one CentOS 6.5 AMI.

Yes, kernel, grub, etc. are included in Core - I verified that all of the packages that used to be in additional_packages get installed.

**Comment by djmitche** [comment]
I'm going to land this now - the master part of things works fine, and I'll test the slave part by hand now, before it runs automatically overnight.

**Comment by djmitche** [comment]
On rail's advice I removed the old ami_configs in bbcf97191f21ba2135db5af59ce0a639a71e1dd1

**Pull request merged**